R/install.R

```diff
@@ -126,7 +126,15 @@ R <- function(options, path = tempdir(), env_vars = NULL, ...) {
     on.exit(set_path(old))
   }
 
-  in_dir(path, system_check(r_path, options, c(r_env_vars(), env_vars), ...))
+  in_dir(
+    path,
+    system_check(
+      cmd = r_path,
+      args = options,
+      env = c(r_env_vars(), env_vars, envvar_mask()),
```

Comment:
No caller passes `env_vars` to `R(...)`. Should we ditch that argument?

Reply:
Yes, I'm all for ditching unnecessary complexity :)